src/shared/components/palette/fixed-list/write-fixed-list-field.component.ts

```diff
@@ -15,38 +11,14 @@ export class WriteFixedListFieldComponent extends AbstractFieldWriteComponent im
   fixedListControl: FormControl;
 
   ngOnInit() {
-    this.instantiate();
     let notEmpty = this.caseField.value !== null && this.caseField.value !== undefined;
-    this.fixedListControl = this.registerControl(new FormControl(notEmpty ? this.caseField.getValue() : null));
-
+    this.fixedListControl = this.registerControl(new FormControl(notEmpty ? this.caseField.value : null));
+    if (this.hasSanitisedDynamicListData()) {
```

Comment:
did we find out why this is needed?

Reply:
For some reason its not erasing the history